src/services/itemLogin/test/index.test.ts

```diff
@@ -477,6 +479,40 @@ describe('Item Login Tests', () => {
         expect(res.statusCode).toBe(StatusCodes.NOT_FOUND);
       });
     });
+
+    describe('ItemLogin from child', () => {
+      beforeEach(async () => {
+        ({ app } = await build({ member: null }));
+      });
+
+      it('Successfully register when item login is defined in parent', async () => {
+        const payload = USERNAME_LOGIN;
+        // pre-create pseudonymized data
+        const item = await testUtils.saveItem({});
+        const { itemLoginSchema } = await saveItemLoginSchema({
+          item: item as unknown as DiscriminatedItem,
+        });
+        const child = await testUtils.saveItem({ parentItem: item });
+
+        const res = await app.inject({
+          method: HttpMethod.Post,
+          url: `${ITEMS_ROUTE_PREFIX}/${child.id}/login`,
+          payload,
+        });
+
+        const member = res.json();
+
+        // membership is saved on the right path
+        const membership = await rawItemMembershipRepository.findOne({
+          where: { account: { id: member.id } },
+          relations: { item: true, account: true },
+        });
+        expect(membership?.item.path).toEqual(itemLoginSchema.item.path);
+
+        expect(res.statusCode).toBe(StatusCodes.OK);
```

Comment:
Test this before using the response ? if the status code is not OK, it is likely that the json will not be a member.